nautobot/dcim/elevations.py

```diff
@@ -217,7 +223,77 @@ def merge_elevations(self, face):
 
         return elevation
 
-    def render(self, face, unit_width, unit_height, legend_width):
+    def render_csv(self, face: Literal["front", "rear"]) -> str:
+        @dataclass
+        class DeviceCSVRepresentation:
+            name: Optional[int]
+            unit: str
+            device: "Device"
+
+        def _get_face_data(face: Literal["front", "rear"]) -> list[DeviceCSVRepresentation]:
+            # Prepare an empty rack representation
+            rack_positions = [None for x in range(0, self.rack.u_height)]
+
+            # Fill with used rack units
+            for unit in self.rack.get_rack_units(face=face, expand_devices=True):
+                if unit["device"] is None:
+                    rack_positions[unit["id"] - 1] = DeviceCSVRepresentation(name="", unit=unit["name"], device=None)
+
+                else:
+                    rack_positions[unit["id"] - 1] = DeviceCSVRepresentation(
+                        name=unit["device"].name,
+                        unit=unit["name"],
+                        device=unit["device"],
+                    )
```

Comment:
Would something like the below work just as well?

```suggestion
            rack_positions = []

            # Fill with used rack units
            for unit in self.rack.get_rack_units(face=face, expand_devices=True):
                if unit["device"] is None:
                    rack_positions.append(DeviceCSVRepresentation(name="", unit=unit["name"], device=None))

                else:
                    rack_positions.append(DeviceCSVRepresentation(
                        name=unit["device"].name,
                        unit=unit["name"],
                        device=unit["device"],
                    ))
```

I think this would also preserve the `desc_units` ordering of the given rack, simplifying `_output_face_to_csv` below as well?

Reply:
Yes... and no.
In case we have to export multiple racks side-by-side (this is a requested feature) and those racks are of different sizes, we can simply find the tallest rack in the row and start requesting directly by unit.

But then again, as I type, I realise we can just index by position and skip that rack should it not be tall enough.
What do you think?

Reply:
I'm not sure how this code would need to change to export multiple racks in a single CSV, to be honest.

Reply:
Another option could be to return two different rows, one for the front and one for the back view.

This way, should we produce a "row" view, we could just ´itertools.zip_longest()´over the entire row for both front and back and we can "ground" the racks on the same line.

What do you think?